Auto is ready for testing

maddyp7 opened the pull request:
- Made lots of paths and autos
- created the framework to run the auto paths
- added network tables entry to choose autos

maddyp7 committed:
Sussex paths

still need to test

NoahRen25 committed:
moved lightgate to transfer

GavinArchon committed:
WIP

komei, liam, gavin

maddyp7 committed:
Ready for testing

Ignore autoCommand.java

KomeiOkamoto12390 committed:
WIP

Simple command
TEST THIS TO MAKE SURE THIS WORKS IF COMPLICATED CODE IS NOT WORKING

maddyp7 committed:
Added auto chooser

aaron97neu commented:
We should be _very_ careful of this caveat: 
> This method will load all autos in the deploy directory. Since the deploy process does not automatically clear the deploy directory, old auto files that have since been deleted from the project could remain on the RIO, therefore being added to the auto chooser.
>
>To remove old options, the deploy directory will need to be cleared manually via SSH, WinSCP, reimaging the RIO, etc.


https://pathplanner.dev/pplib-build-an-auto.html#create-a-sendablechooser-with-all-autos-in-project

aaron97neu commented:
May consider adding this as a example for triggering commands from auto

https://pathplanner.dev/pplib-named-commands.html

NoahRen25 committed:
wip

NoahRen25 committed:
tell me everything wrong (a lot) WIP

dlaufen committed:
Minor code tweeks, better names for a few vars.

KomeiOkamoto12390 committed:
auto shooting command

Bug found in FaceToTag

KomeiOkamoto12390 committed:
Fix and new feature

- bug fix
- switch between scoring targets
- Changing angle and RPM not implemented yet

NoahRen25 committed:
Shooter Sequence

NoahRen25 committed:
Limit switch on motor controller

maddyp7 committed:
Added commands

KomeiOkamoto12390 committed:
Fix

KomeiOkamoto12390 committed:
Merge branch 'FaceToTagAndShooter' of https://github.com/2202Programming/FRC-2024 into FaceToTagAndShooter

KomeiOkamoto12390 committed:
WIP

NoahRen25 committed:
pls check this!! gear ratio stuff ig

KomeiOkamoto12390 committed:
Clean up

KomeiOkamoto12390 committed:
note

NoahRen25 committed:
testing

NoahRen25 committed:
forgot to save lol

NoahRen25 committed:
changes

NoahRen25 committed:
driver station laptop no push!?!?!??!

NoahRen25 committed:
Merge branch 'ShooterPneumatics' into SussexMain

NoahRen25 committed:
merged

NoahRen25 committed:
e

maddyp7 committed:
updating branch

dlaufen reviewed (approved):
looks good.  Aaron is right about the rio.  Need to watch that.
